plugin/trino-redshift/src/test/java/io/trino/plugin/redshift/BaseRedshiftCastPushdownTest.java

```diff
@@ -59,16 +59,25 @@ public void setupTable()
                 .addColumn("c_numeric_10_2", "numeric(10, 2)", asList(1.23, 2.67, null))
                 .addColumn("c_numeric_19_2", "numeric(19, 2)", asList(1.23, 2.67, null)) // Equal to REDSHIFT_DECIMAL_CUTOFF_PRECISION
                 .addColumn("c_numeric_30_2", "numeric(30, 2)", asList(1.23, 2.67, null))
+                .addColumn("c_char", "char", asList("'A'", "'B'", null))
+                .addColumn("c_character", "character", asList("'A'", "'B'", null))
```

Comment:
Can we have same data as other char and varchar columns.

Reply:
Its because char/character without precision is char(1) https://docs.aws.amazon.com/redshift/latest/dg/r_Character_types.html#r_Character_types-char-or-character